content/company.md

```diff
@@ -6,4 +6,12 @@ draft: false
 
 Qri is a 5-person startup lucky enough to work with a _wonderful_ community of friends. We do work in Brooklyn, NY.
 
+We are:
```

Comment:
This is lovely! In the future it'd be great if this was structured as a separate PR